go/signedexchange/README.md

```diff
@@ -1,3 +1,83 @@
-# gen-signedexchange
+# go/bundle
+This directory contains a reference implementation of [Signed HTTP Exchanges](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html) spec.
 
-This directory will hold a reference implementation of a tool to sign exchanges, but for now, the version that best matches Chrome is in https://github.com/nyaxt/webpackage/tree/master/go/signedexchange .
+## Overview
+We currently provide two command-line tools: `gen-signedexchange` and `gen-certurl`.
+
+`gen-signedexchange` generates a signed exchange file. The `gen-signedexchange` command constructs HTTP request and response pair from given command line flags, attach a cryptographic signature of the pair, and serializes the result to output file.
+
+`gen-certurl` converts a X.509 certificate chain to `application/cert-chain+cbor` format, which is defined in the [Section 3.3 of the Signed HTTP Exchanges spec](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#rfc.section.3.3).
+
+You are also welcome to use the code as golang lib (e.g. `import "github.com/WICG/webpackage/go/signedexchange"`), but please be aware that the API is not yet stable and is subject to change any time.
+
+## Getting Started
+
+### Prerequisite
+golang environment needs to be set up in prior to using the tool. We are testing the tool on latest golang. Please refer to [Go Getting Started documentation](https://golang.org/doc/install) for the details.
+
+### Installation
+We recommend using `go get` to install the command-line tool.
+
+```
+go get -u github.com/WICG/webpackage/go/signedexchange/cmd/...
+```
+
+### Creating our first signed exchange
+In this section, we guide you to create a signed exchange file, signed using self-signed certificate pair.
+
+Here, we assume that you have an access to a HTTPS server capable of serving static content. [1] Please substitute `https://yourcdn.example.net/` URLs to your web server URL.
```

Comment:
an HTTPS server

static content -> static contents

Reply:
I would keep "static content". I think that's because "content" is used as a [mass noun](https://en.wikipedia.org/wiki/Mass_noun) for internet content, even though it can turn into "contents" when describing the contents of a box (but the content of a box would be fine too ... if you're thinking of all of the content as one notional thing ...). Sorry about English.

Reply:
Did s/a/an